CMakeLists.txt

```diff
@@ -120,6 +120,31 @@ include(CPack)
 # tests
 #
 
+add_custom_command(
+  OUTPUT test_private.pem
+  COMMAND openssl genrsa -out test_private.pem -3 3072)
+
+# build the test marble with ertgo and sign it with oesign
+add_custom_command(
+  OUTPUT test-marble
+  DEPENDS ego-enclave test_private.pem cmd/test-marble/enclave.conf cmd/test-marble/main.go internal/test/t.go
+  COMMAND ${CMAKE_COMMAND} -E env GOROOT=${CMAKE_SOURCE_DIR}/_ertgo ${CMAKE_SOURCE_DIR}/_ertgo/bin/go build -o ${CMAKE_BINARY_DIR}
+  COMMAND oesign sign
```

Comment:
I don't understand this quite yet. Why use ertgo & oesign over ego-go and ego sign?

Reply:
Because the latter don't work unless they are installed. But it's more or less the same what these tools would execute.

Reply:
Well... of course testing ego directly would be better because ego is what we want to test here after all and we tell the user to use ego to sign things, not oesign... But if it's not possible easily, guess we can just keep it that way.

Reply:
Agreed that it would be better. But actually, I want to test only ego-enclave here, not the CLI tool. That's covered in other tests.

Reply:
Well, if that's the case, it's fine for me :)